### PR TITLE
DOC: Fixing EX01 - Added examples

### PR DESCRIPTION
**Pull request by DeaMariaLeon**
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).

Towards https://github.com/pandas-dev/pandas/issues/37875

### CODE REVIEW

**Comment by DeaMariaLeon** [comment]
It seems that I'm bugging 2 reviewers! But I'm requesting Marco's review because we talked about it... The request for Matthew's review happens automatically now.

File: pandas/core/strings/accessor.py [hunk 1]
```diff
@@ -1616,6 +1634,35 @@ def pad(
...
+    dtype: object
+
+    For Series.str.rjust:
+
+    >>> ser = pd.Series(['dog', 'bird', 'mouse'])
+    >>> ser.str.rjust(8, fillchar='.')
+    0   .....dog
+    1   ....bird
+    2   ...mouse
+    dtype: object
```
Comment (MarcoGorelli) [inline pandas/core/strings/accessor.py:1665]: today I learned about this functionality..

**Review by MarcoGorelli — APPROVED** [review]
nice, thanks @DeaMariaLeon ! I didn't even know that these functions exited :smile:

**Comment by MarcoGorelli** [comment]
> The request for Matthew's review happens automatically now.

it's because of the codeowners file that's recently been added, so he gets notified for any PR touching `ci`

**Comment by DeaMariaLeon** [comment]
Thanks @MarcoGorelli

**Pull request merged**